src/binary-reader-linker.cc

```diff
@@ -243,11 +261,16 @@ Result BinaryReaderLinker::OnExport(Index index,
                                     ExternalKind kind,
                                     Index item_index,
                                     StringSlice name) {
+  if (kind == ExternalKind::Memory) {
+    WABT_FATAL("Linker does not support exported memories");
+  }
+
```

Comment:
I guess I'm not really sure why we shouldn't just reject memory imports like this too.   Wouldn't that be a lot simpler?   What is the advantage of supporting memory imports in the input objects?